manifests/service.pp

```diff
@@ -4,6 +4,12 @@
 #
 class redis::service {
   if $::redis::service_manage {
+    file {
+      $::redis::service_file_orig:
```

Comment:
Wouldn't this mean that the systemd unit file is always added, even if the user is not on a systemd system?